maker/src/cfd.rs

```diff
@@ -525,44 +528,71 @@ impl<O, T, W> Actor<O, T, W> {
 
         let order_id = msg.order_id;
 
-        match self
-            .rollover_actors
-            .send_async(
-                &order_id,
-                rollover::AcceptRollover {
+        let cfd = self.db.load_open_cfd::<Cfd>(order_id, ()).await?;
+
+        if cfd.counterparty_peer_id().is_some() {
+            // Using send here is fine because we dispatch to a task internally
+            self.libp2p_rollover
+                .send(daemon::rollover::maker::Accept {
```

Comment:
❓ Do we need to though?

Reply:
The original idea was to be able to give *some* error information back to the HTTP endpoint. But I don't think the caller actually cares. 
The only error that is currently returned is an error where we don't have a pending rollover internally. This could be useful for the caller at some point. But for now probably not. 
I'm happy to change this to dispatch async, but then I'd say we should change the `Accept` handler *not* to return a `Result`. It does not make sense to return a `Result` if we don't care about it. @klochowicz what do you think?